src/components/quickstarts/QuickstartOverview.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { css } from '@emotion/react';
+
+import { quickstart } from '../../types';
+import OverviewTile from './OverviewTile';
+
+const QuickstartOverview = ({ quickstart }) => {
+  return (
+    <>
+      <h3> What&apos;s included: </h3>
+      <div
+        css={css`
+          display: grid;
+          grid-gap: 1rem;
+          grid-template-columns: repeat(3, 1fr);
+
+          @media (max-width: 1180px) {
+            grid-template-columns: repeat(1, 1fr);
+          }
+        `}
+      >
+        {quickstart.dashboards.map((dashboard, index) => (
+          <OverviewTile
+            key={index}
+            title={dashboard.name}
+            image={dashboard?.screenshots[0]}
```

Comment:
i'm thinking you might want `dashboard.screenshots?.[0]`. having the optional chaining after dashboards would resolve if dashboards is undefined but i think maybe this is to check that screenshots exist? also, i believe all fields return an empty array now instead of null but the optional doesn't hurt

Reply:
whoops 🙈

Reply:
I forget, do we have access to optional chaining in this repo?

Reply:
oh good point, no i think it's on node 12 😂

Reply:
removed. thats another good point for my whoops 🙈